auto_install/install.sh

```diff
@@ -527,12 +578,37 @@ preconfigurePackages(){
 		USING_UFW=0
 	fi
 
-	if [ "$USING_UFW" -eq 0 ]; then
+	if [ "${PKG_MANAGER}" == 'apt-get' ] && [ "$USING_UFW" -eq 0 ]; then
 		BASE_DEPS+=(iptables-persistent)
 		echo iptables-persistent iptables-persistent/autosave_v4 boolean true | $SUDO debconf-set-selections
 		echo iptables-persistent iptables-persistent/autosave_v6 boolean false | $SUDO debconf-set-selections
 	fi
 
+	# install grepcidr manually, if not present
+	if [[ "${PLAT}" == 'Alpine' ]] && ! command -v grepcidr &> /dev/null; then
+		## install dependencies
+		${PKG_INSTALL} build-base make curl tar
+
+		## download binaeries
+		curl -fLo master.tar.gz https://github.com/pivpn/grepcidr/archive/master.tar.gz
+		tar -xzf master.tar.gz
+
+		## personalize binaries
+		sed -i -E -e 's/^PREFIX\=.*/PREFIX\=\/usr\nCC\=gcc/' grepcidr-master/Makefile
+
+		## install
+		(
+			cd grepcidr-master || exit
+
+			make
+			make install
+		)
+
+		## remove useless files
+		rm master.tar.gz
+		rm -rf grepcidr-master
+	fi
```

Comment:
- Need sudo to install packages
- Enter the `grepcidr-master` folder (thus implying its existence) before editing `Makefile`
- `make install` needs sudo too
- Verify that grepcidr was installed
- Compile in randomly generated folder in case `master.tar.gz` exists in the current working directory maybe too?

```suggestion
	# install grepcidr manually, if not present
	if [[ "${PLAT}" == 'Alpine' ]] && ! command -v grepcidr &> /dev/null; then
		## install dependencies
		${SUDO} ${PKG_INSTALL} build-base make curl tar

		## download source code
		curl -fLo master.tar.gz https://github.com/pivpn/grepcidr/archive/master.tar.gz
		tar -xzf master.tar.gz

		cd grepcidr-master || exit 1

		## personalize binaries
		sed -i -E -e 's/^PREFIX\=.*/PREFIX\=\/usr\nCC\=gcc/' Makefile

		## compile and install
		make
		${SUDO} make install

		if ! command -v grepcidr &> /dev/null; then
			echo "::: Failed to compile and install grepcidr!"
			exit 1
		fi

		## remove useless files
		rm master.tar.gz
		rm -rf grepcidr-master
	fi

```